docs/api/api/api.txt

```diff
@@ -1847,6 +1847,36 @@ PUT /message/?project=<project>&package=<package>
 XmlBody :message
 XmlResult: message
 
+== Status Checks
+  WARNING: This API is currently in beta and unstable
+
+=== Repositories
+
+<project>: The name of a project, e.g. home:user1
+<repository>: The name of a repository of the project, e.g. openSUSE_Tumbleweed
+<uuid>: The unique identifier you have received for the Repository
+
+GET /status_reports/published/<project>/<repository>/reports/<uuid>
+  Get a status report with this uuid
+XmlResult: status_report
+
+
+PUT /status_reports/published/<project>/<repository>/reports/<uuid>/checks
+  Create or update check data
+XmlResult: check
+
+=== Requests
+
+<request>: The number of the request, e.g. 12345
+
+GET /status_reports/requests/<request>/reports
+  Get a the status report for this request
+XmlResult: status_report
+
+
+PUT /status_reports/requests/<request>/reports/checks
+  Create or update check data
+XmlResult: check
```

Comment:
you upload multiple checks and get one check back?

Reply:
Nope I upload one check. Get one back.